src/app/component-status/component-status-items.ts

```diff
@@ -240,6 +240,19 @@ export const componentStatusItems: ComponentStatusItem[] = [
       cookbookUrl: 'showcase/chip',
     },
   },
+  {
+    title: 'Empty State',
+    priority: 0,
+    ux: {
+      version: 0.0,
+      status: ItemUXStatus.underConsideration,
+    },
+    code: {
+      version: 0.1,
```

Comment:
Probably should be closer to 1.0 (e.g. 0.9)

Reply:
It is the first version of the component so I think 0.1 is more appropriate. It is not the ninth version of it.

Reply:
0.8

Reply:
0.8 it is